lib/src/util/vulnerabilitySeverityLevels.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+/**
+ * All Possible Vulnerability (CVSS) Severity Levels.
+ */
+export const allVulnerabilitySeverityLevels = [
```

Comment:
I recommend using the official terminology for these.  They are "ratings", not "levels".  See https://nvd.nist.gov/vuln-metrics/cvss

Reply:
Solved in 850e0f97fb067d53160d6492597a98ad374cc162